### ISSUE TITLE
Dropdown control in multi-select mode does not have a role.

### ISSUE THREAD

**Issue by SerhiiYatsenko** [OfficeDev/office-ui-fabric-react#11657]
### Environment Information

- **Package version(s)**: latest
- **Browser and OS versions**: Chrome, Windows 10

### Describe the issue:
If a [Dropdown](https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown) control in multi-select mode it does not have a role ([line 231](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx)). So LocalizedControlType is null.

This accessibility issue was found using **Accessibility Insights for Windows**, a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at https://aka.ms/AccessibilityInsights.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/serhiiyatsenko/pen/dyPmzaY?editable=true (or see Multi-select controlled example  [here](https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown))

#### Actual behavior:
The LocalizedControlType property does not exist.

#### Expected behavior:
The LocalizedControlType property must not be null.

### Documentation describing expected behavior
 [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

> 502.3.1 Object Information. The object role, state(s), properties, boundary, name, and description shall be programmatically determinable.

**Comment by xugao** [comment]
looks like a bug. multi-select combobox should have `aria-multiselectable="true"` and `listbox` role. i wonder what's the history behind the control not having them

**Comment by xugao** [comment]
@SerhiiYatsenko - can you tell me a bit more how you tested using the tool? I run a FastPass test in Chrome on https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown, which renders the multi-select dropdown, but it didn't complain about this missing role.

**Comment by SerhiiYatsenko** [comment]
@xugao 
1. Run Accessibility Insights (Live Inspect).
2. Use tab key to navigate to the DropDown control.
3. Click a test button.
(please see the attached gif below)
![dropdownrole](https://user-images.githubusercontent.com/16796085/72133796-007ec480-338b-11ea-96a2-0991ff0dfdd0.gif)

You can open the attached Accessibility Insights for Windows Test File (*.a11ytest) to reproduce the error in Accessibility Insights for Windows. 
[DropDownTestResults 01-10-20.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/4044413/DropDownTestResults.01-10-20.zip)
